Standards/scs-xxxx-v1-status-page-create-decision.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Status Page create decision
+type: Decision Record
+status: Draft
+track: Ops
+enhances: status-page-comparison.md
+---
+
+# Introduction
+
+Creating and maintaining IaaS is a complex task. Any kind of consumer (e.g. operators, cutsomers) can
+be supported by presenting the status of all possible parts of the
+serving infrastructure. Whether a service is not reachable or
+the used hardware is having an outage we want the consumers to be easily informed
+by using a "Status Page" application. The need for a "Status Page"
+came up early in Team OPS and the requirements a "Status Page" application 
+has to fulfill were defined and written down on 2022-06-02 as a 
+[MVP-0 epic](https://github.com/SovereignCloudStack/issues/issues/123).
+The upcoming research on existing solutions came to the conclusion that we want to
+create a new "Status Page" application.
+
+# Existing Applications
+
+Since we want to use as much as possible from existing projects and contribute to
+upstream projects to support the community with our efforts, it was a hard
+decision to create a new "Status Page" application.
+
+Before the decision was made some existing and known applications were tested
+and analyzed if they would fit to our use case. An overview of this
+comparsion can be found [here](https://github.com/SovereignCloudStack/Docs/blob/main/Decisions/status-page-comparison.md)
```

Comment:
This link does no longer work unfortunately.
`s/Docs/standards/` to make it work.